erts/emulator/test/distribution_SUITE.erl

```diff
@@ -2608,7 +2614,7 @@ port_please(_Name, _Ip) ->
 	    {port, Port, Version}
     end.
 
-address_please(_Name, _Address, _AddressFamily) ->
+address_please(_Name, "dummy", _AddressFamily) ->
```

Comment:
Would it not make sense to make the fallback consider the AdressFamily? Maybe this test is never run over ipv6?

Reply:
hmm, good question.